*** Starting uWSGI 2.0.18 (64bit) on [Mon May  6 03:40:45 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x55ad55d153b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55ad55d153b0 pid: 583 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 583)
spawned uWSGI worker 1 (pid: 645, cores: 1)
spawned uWSGI worker 2 (pid: 646, cores: 1)
spawned uWSGI worker 3 (pid: 647, cores: 1)
spawned uWSGI worker 4 (pid: 648, cores: 1)
spawned uWSGI worker 5 (pid: 649, cores: 1)
Python auto-reloader enabled
[pid: 646|app: 0|req: 1/1] 10.0.2.2 () {40 vars in 730 bytes} [Mon May  6 03:41:02 2019] GET / => generated 1161 bytes in 24 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 649|app: 0|req: 1/2] 10.0.2.2 () {40 vars in 676 bytes} [Mon May  6 03:41:02 2019] GET /favicon.ico => generated 232 bytes in 32 msecs (HTTP/1.1 404) 2 headers in 72 bytes (1 switches on core 0)
[pid: 646|app: 0|req: 2/3] 10.0.2.2 () {42 vars in 833 bytes} [Mon May  6 03:41:06 2019] GET /search?query_type=or&query=orange => generated 775 bytes in 17 msecs (HTTP/1.1 200) 2 headers in 80 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Mon May  6 03:42:15 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x55a8dd2dd3b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x55a8dd2dd3b0 pid: 704 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 704)
spawned uWSGI worker 1 (pid: 706, cores: 1)
spawned uWSGI worker 2 (pid: 707, cores: 1)
spawned uWSGI worker 3 (pid: 708, cores: 1)
spawned uWSGI worker 4 (pid: 709, cores: 1)
spawned uWSGI worker 5 (pid: 710, cores: 1)
Python auto-reloader enabled
[uwsgi-python-reloader] module/file /home/cs143/www/SearchEngine/SearchEngine/searchengine.py has been modified
...gracefully killing workers...
Gracefully killing worker 1 (pid: 706)...
Gracefully killing worker 2 (pid: 707)...
Gracefully killing worker 3 (pid: 708)...
Gracefully killing worker 4 (pid: 709)...
Gracefully killing worker 5 (pid: 710)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
binary reloading uWSGI...
chdir() to /media/sf_vm-shared/SearchEngine
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/tmp/searchengine.sock)
running /home/cs143/.local/bin/uwsgi
[uWSGI] getting INI configuration from /home/cs143/www/SearchEngine/searchengine.ini
*** Starting uWSGI 2.0.18 (64bit) on [Mon May  6 03:42:33 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x55efcb3eb330
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55efcb3eb330 pid: 704 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 704)
spawned uWSGI worker 1 (pid: 718, cores: 1)
spawned uWSGI worker 2 (pid: 719, cores: 1)
spawned uWSGI worker 3 (pid: 720, cores: 1)
spawned uWSGI worker 4 (pid: 721, cores: 1)
spawned uWSGI worker 5 (pid: 722, cores: 1)
Python auto-reloader enabled
Traceback (most recent call last):
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cs143/www/SearchEngine/SearchEngine/searchengine.py", line 16, in dosearch
    search_results, result_len = search.search(query, qtype, offset)
TypeError: search() takes 2 positional arguments but 3 were given
[pid: 719|app: 0|req: 1/1] 10.0.2.2 () {42 vars in 833 bytes} [Mon May  6 03:42:36 2019] GET /search?query_type=or&query=orange => generated 0 bytes in 40 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[pid: 719|app: 0|req: 2/2] 10.0.2.2 () {42 vars in 754 bytes} [Mon May  6 03:43:13 2019] GET / => generated 1161 bytes in 29 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
Traceback (most recent call last):
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/cs143/www/SearchEngine/SearchEngine/searchengine.py", line 16, in dosearch
    search_results, result_len = search.search(query, qtype, offset)
TypeError: search() takes 2 positional arguments but 3 were given
[pid: 719|app: 0|req: 3/3] 10.0.2.2 () {42 vars in 833 bytes} [Mon May  6 03:43:16 2019] GET /search?query_type=or&query=orange => generated 0 bytes in 25 msecs (HTTP/1.1 500) 0 headers in 0 bytes (0 switches on core 0)
[uwsgi-python-reloader] module/file /home/cs143/www/SearchEngine/SearchEngine/search.py has been modified
...gracefully killing workers...
Gracefully killing worker 1 (pid: 718)...
Gracefully killing worker 2 (pid: 719)...
Error in atexit._run_exitfuncs:
Gracefully killing worker 3 (pid: 720)...
Gracefully killing worker 4 (pid: 721)...
Gracefully killing worker 5 (pid: 722)...
Traceback (most recent call last):
  File "/usr/lib/python3.6/logging/__init__.py", line 1945, in shutdown
    h.flush()
  File "/usr/lib/python3.6/logging/__init__.py", line 977, in flush
    if self.stream and hasattr(self.stream, "flush"):
  File "/home/cs143/.local/lib/python3.6/site-packages/werkzeug/local.py", line 329, in __bool__
    return bool(self._get_current_object())
  File "/home/cs143/.local/lib/python3.6/site-packages/werkzeug/local.py", line 307, in _get_current_object
    return self.__local()
  File "/home/cs143/.local/lib/python3.6/site-packages/flask/logging.py", line 30, in wsgi_errors_stream
    return request.environ['wsgi.errors'] if request else sys.stderr
KeyError: 'wsgi.errors'
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
binary reloading uWSGI...
chdir() to /media/sf_vm-shared/SearchEngine
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (/tmp/searchengine.sock)
running /home/cs143/.local/bin/uwsgi
[uWSGI] getting INI configuration from /home/cs143/www/SearchEngine/searchengine.ini
*** Starting uWSGI 2.0.18 (64bit) on [Mon May  6 03:43:45 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x55635b985330
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x55635b985330 pid: 704 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 704)
spawned uWSGI worker 1 (pid: 738, cores: 1)
spawned uWSGI worker 2 (pid: 739, cores: 1)
spawned uWSGI worker 3 (pid: 740, cores: 1)
spawned uWSGI worker 4 (pid: 741, cores: 1)
spawned uWSGI worker 5 (pid: 742, cores: 1)
Python auto-reloader enabled
[pid: 739|app: 0|req: 1/1] 10.0.2.2 () {42 vars in 754 bytes} [Mon May  6 03:43:50 2019] GET / => generated 1161 bytes in 36 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 739|app: 0|req: 2/2] 10.0.2.2 () {42 vars in 831 bytes} [Mon May  6 03:43:56 2019] GET /search?query_type=or&query=hello => generated 3382 bytes in 8470 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 739|app: 0|req: 3/3] 10.0.2.2 () {42 vars in 863 bytes} [Mon May  6 03:44:29 2019] GET /search?query_type=or&query=hello => generated 3505 bytes in 2812 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 739|app: 0|req: 4/4] 10.0.2.2 () {42 vars in 865 bytes} [Mon May  6 03:44:35 2019] GET /search?query_type=and&query=hello => generated 3506 bytes in 2683 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Thu May 16 20:24:18 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x561eba1683b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x561eba1683b0 pid: 582 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 582)
spawned uWSGI worker 1 (pid: 644, cores: 1)
spawned uWSGI worker 2 (pid: 645, cores: 1)
spawned uWSGI worker 3 (pid: 646, cores: 1)
spawned uWSGI worker 4 (pid: 647, cores: 1)
spawned uWSGI worker 5 (pid: 648, cores: 1)
Python auto-reloader enabled
*** Starting uWSGI 2.0.18 (64bit) on [Thu May 16 20:25:48 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x5569f98e33b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x5569f98e33b0 pid: 714 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 714)
spawned uWSGI worker 1 (pid: 716, cores: 1)
spawned uWSGI worker 2 (pid: 717, cores: 1)
spawned uWSGI worker 3 (pid: 718, cores: 1)
spawned uWSGI worker 4 (pid: 719, cores: 1)
spawned uWSGI worker 5 (pid: 720, cores: 1)
Python auto-reloader enabled
[pid: 719|app: 0|req: 1/1] 10.0.2.2 () {42 vars in 754 bytes} [Thu May 16 22:31:35 2019] GET / => generated 1161 bytes in 38 msecs (HTTP/1.1 200) 2 headers in 81 bytes (1 switches on core 0)
[pid: 717|app: 0|req: 1/2] 10.0.2.2 () {42 vars in 703 bytes} [Thu May 16 22:31:35 2019] GET /static/searchengine.css => generated 457 bytes in 38 msecs via sendfile() (HTTP/1.1 200) 8 headers in 299 bytes (0 switches on core 0)
[pid: 720|app: 0|req: 1/3] 10.0.2.2 () {42 vars in 712 bytes} [Thu May 16 22:31:35 2019] GET /static/header.png => generated 14149 bytes in 39 msecs via sendfile() (HTTP/1.1 200) 8 headers in 289 bytes (0 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Mon May 20 21:47:40 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x56329bdda3b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x56329bdda3b0 pid: 587 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 587)
spawned uWSGI worker 1 (pid: 644, cores: 1)
spawned uWSGI worker 2 (pid: 645, cores: 1)
spawned uWSGI worker 3 (pid: 646, cores: 1)
spawned uWSGI worker 4 (pid: 647, cores: 1)
spawned uWSGI worker 5 (pid: 648, cores: 1)
Python auto-reloader enabled
*** Starting uWSGI 2.0.18 (64bit) on [Mon May 20 21:50:01 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x56369605d3b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x56369605d3b0 pid: 620 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 620)
spawned uWSGI worker 1 (pid: 681, cores: 1)
spawned uWSGI worker 2 (pid: 682, cores: 1)
spawned uWSGI worker 3 (pid: 683, cores: 1)
spawned uWSGI worker 4 (pid: 684, cores: 1)
spawned uWSGI worker 5 (pid: 685, cores: 1)
Python auto-reloader enabled
*** Starting uWSGI 2.0.18 (64bit) on [Mon May 20 21:51:31 2019] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x55bd631be3b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55bd631be3b0 pid: 739 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 739)
spawned uWSGI worker 1 (pid: 742, cores: 1)
spawned uWSGI worker 2 (pid: 743, cores: 1)
spawned uWSGI worker 3 (pid: 744, cores: 1)
spawned uWSGI worker 4 (pid: 745, cores: 1)
spawned uWSGI worker 5 (pid: 746, cores: 1)
Python auto-reloader enabled
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: unix socket /tmp/searchengine.sock removed.
*** Starting uWSGI 2.0.18 (64bit) on [Sun May 10 18:04:01 2020] ***
compiled with version: 7.3.0 on 30 March 2019 03:07:46
os: Linux-4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019
nodename: cs143-19s-p1
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /media/sf_vm-shared/SearchEngine
detected binary path: /home/cs143/.local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
your processes number limit is 7804
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /tmp/searchengine.sock fd 3
Python version: 3.6.7 (default, Oct 22 2018, 11:32:17)  [GCC 8.2.0]
Python main interpreter initialized at 0x55bc5fa503b0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 437520 bytes (427 KB) for 5 cores
*** Operational MODE: preforking ***
